src/main/java/com/google/jenkins/plugins/googlecontainerregistryauth/GoogleContainerRegistryCredential.java

```diff
@@ -116,7 +116,7 @@ public GoogleRobotCredentials getCredentials() {
    * serialize things.
    */
   private boolean isOnMaster() {
-    return Jenkins.getInstance() != null;
+    return Jenkins.getInstanceOrNull() != null;
```

Comment:
Don't do the above, it can cause errors on the agent when it trys to transfer the Jenkins class to the agent

```suggestion
    return JenkinsJVM.isJenkinsJVM();
```